src/main/resources/META-INF/services/org.apache.pirk.responder.wideskies.spi.ResponderPlugin

```diff
@@ -0,0 +1,5 @@
+org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponder
```

Comment:
> not only does the user have to specify the class/platform

Yes.  Do you envisage some way in which the user could **not** have to specify the platform they want Pirk to use?

> the code has to be aware, via a static list, of the possible plugin
> implementations (as listed in org.apache.pirk.responder.wideskies.spi.ResponderPlugin)

The static list of plugins is in the ```META-INF/services file``` that lists all the known implementers of that interface _in this JAR_.  Today, because we have one large JAR, all the responders are listed, but (i) when we split into submodules each impl will move to its corresponding JAR artefact, and (ii) anyone can now drop a JAR on the classpath with an impl declared in it's services list, and Pirk will find it by getPlatformName.

How do you propose pirk-core finds the responder implementers at runtime?